faqs/galaxy/datasets_finding_datasets.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Finding Datasets
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+- To review all active Datasets in your account, go to **User > Datasets**.
```

Comment:
I'd say that the FAQ is just this first step and the rest are notes, what do you think @Melkeb?

Reply:
Yeah you are right